modules/loader/src/test/resources/com/opengamma/strata/loader/csv/trades-cpty.csv

```diff
@@ -0,0 +1,23 @@
+Strata Trade Type,Id Scheme,Id,Counterparty Scheme,Counterparty,Trade Date,Trade Time,Trade Zone,Convention,Buy Sell,Period To Start,Tenor,Index,Interpolated index,Fixed Rate,FX Rate,Day Count,Currency,Notional,Start Date,End Date,Date Convention,Date Calendar,Stub Convention,Leg 1 Direction,Leg 1 Start Date,Leg 1 End Date,Leg 1 Stub Convention,Leg 1 Roll Convention,Leg 1 Frequency,Leg 1 Currency,Leg 1 Notional,Leg 1 Fixed Rate,Leg 1 Day Count,Leg 1 Date Convention,Leg 1 Date Calendar,Leg 2 Direction,Leg 2 Start Date,Leg 2 End Date,Leg 2 Frequency,Leg 2 Currency,Leg 2 Notional,Leg 2 Index,Security Id Scheme,Security Id,Quantity,Long Quantity,Price
+Fra,OG,123401,CPTY,Bank A,01/06/2017,11:05,Europe/London,GBP-LIBOR-3M,Buy,P2M,,,,0.5,,,,1000000,,,,,,,,,,,,,,,,,,,,,,,,,,,,,

```

Comment:
It would be simpler to add the extra two columns to the existing `trades.csv` file and extend the existing tests, rather than cutting and pasting new ones.

Reply:
Need to check 3 cases: without counterparty column, with only the 'Counterparty' column and with both 'Counterparty' and 'Counterparty Scheme' columns. That means at least 3 files.